ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -218,17 +218,86 @@ class SignalProducerSpec: QuickSpec {
 			}
 		}
 
+		describe("ignoreValues") {
+			it("should not forward values from original producer") {
+				let producer: SignalProducer<Int, NoError> = SignalProducer<Int, NoError>(values: [ 1, 2 ]) |> ignoreValues
```

Comment:
What if you change function declaration [here](https://github.com/ReactiveCocoa/ReactiveCocoa/pull/1719/files#diff-48c01e61977d5d46be16565ecfb017e2R566) from:
```
public func ignoreValues<T, U, E>(producer: SignalProducer<T, E>) -> SignalProducer<U, E>
```
to:
```
public func ignoreValues<T, E>(producer: SignalProducer<T, E>) -> SignalProducer<T, E>
```